recipe/build.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
-$PYTHON setup.py install
+export SETUPTOOLS_DISABLE_VERSIONED_EASY_INSTALL_SCRIPT=0
+export DISTRIBUTE_DISABLE_VERSIONED_EASY_INSTALL_SCRIPT=0
```

Comment:
These flags were created for external package managers. We may still create the `symlink`, but I am not sure if that is needed as `conda` does not allow the installation of two Python versions in the same env.